src/Plugin/resource/DataProvider/DataProviderPlug.php

```diff
@@ -129,10 +129,10 @@ public function remove($identifier) {
    * {@inheritdoc}
    */
   public function getIndexIds() {
-    // Return all of the instance IDs, even for disabled resources.
+    // Return all of the instance IDs.
     $plugins = restful()
       ->getResourceManager()
-      ->getPlugins(TRUE);
+      ->getPlugins();
```

Comment:
@e0ipso I'm changing it here as in `applyFilters` we will get an error for a disabled plugin.